.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: cpp
+
+notifications:
+  email: false
+
+sudo: required
+dist: trusty
+
+branches:
+  only:
+    - master
+    - B3_0_Release
+
+install:
+  # wget --no-check-certificate https://www.cmake.org/files/v3.2/cmake-3.2.3-Linux-x86_64.sh
+  # sudo sh cmake-3.2.3-Linux-x86_64.sh --skip-license --prefix=/usr
+  - sudo apt-get update
+  - sudo apt-get install -y libtommath0 libtommath-dev libtomcrypt0 libtomcrypt-dev
```

Comment:
This libraries are not necessary for v2.5.

Reply:
Thank you , i have removed the libs